doc/ug/dv_methodology/index.md

```diff
@@ -75,7 +76,7 @@ A [template]({{< relref "hw/dv/doc/dv_plan_template" >}}) for the DV plan docume
 ### Regression Dashboard
 
 The DV Plan document provides a link to the latest [nightly](#nightly) regression and coverage results dashboard uploaded to the web server.
-This dashboard contains information in a tabulated format mapping the written tests to planned tests (in the testplan) to provide ability to track progress towards executing the testplan.
+This dashboard contains information in a tabulated format mapping the written tests to planned tests (in the testplan) to provide ability to track progress towards executing the testplan.It likewise contain information mapping the written coverage points in the coverage plan to provide the ability to track progess on hitting the coverage goals.
```

Comment:
This document is one line per sentence, so you need a newline before "It". Also, the verb "contain" should match "it" (singular) so the sentence should start "It likewise contains ...". Finally, there's a missing "r" in "progress".

However, if you're redrafting this anyway, maybe we can make it a bit easier to read. How about:
```
This dashboard contains a table that shows which of the entries in the testplan have associated tests.
The entries with tests also show whether they are passing or not.
The dashboard also contains a table that maps points in the coverage plan to named coverage points in the code.
Entries with such a mapping also show whether the point has been covered or not.
The first table allows the reader to track progress towards executing the testplan.
The second allows the reader to track progress towards executing the coverage plan.
```

Reply:
sure that sounds good to me.